backend/sirius-web-collaborative-diagrams-api/src/main/java/org/eclipse/sirius/web/collaborative/diagrams/api/dto/DeleteFromDiagramInput.java

```diff
@@ -49,16 +53,21 @@ public UUID getRepresentationId() {
         return this.representationId;
     }
 
-    @GraphQLID
     @GraphQLField
     @GraphQLNonNull
-    public String getDiagramElementId() {
-        return this.diagramElementId;
+    public List<@GraphQLNonNull String> getNodeIds() {
```

Comment:
This should be `List<@GraphQLNonNull @GraphQLID String>` or something like that

Reply:
I must update the GraphQLID annotation in this purpose but you are right, I should make this precision.

Reply:
This comment is interesting.
You say that an nodeId is an UUID but the getIdProvider return a String.
If I see the NodeMappingConverter, the idProvider give a string ```UUID.randomUUID().toString()```
(It's a disguised UUID) 

Actually, the UUID implementation of a node ID does not appear in Types :
* Node#id
* NodeDescription#idProvider
* NodeElementProps 
* IDiagramService#findNodeById 

I think that this ```@GraphQLID``` should be add in a dedicated enhancement.